File: Java/프로그래머스/0/181847. 0 떼기/0 떼기.java
class Solution {
    public String solution(String n_str) {
        String answer = "";
        int count=0;
        for(int i=0; i<1;i++){
            for(int j=0; j<n_str.length();j++){
            if(n_str.charAt(j)=='0') count++;
            else break;
            }
        }
        for(int i=count; i<n_str.length();i++){
            answer+=n_str.charAt(i);
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/181847. 0 떼기/0 떼기.js
function solution(n_str) {
    var answer = '';
    var arr = new Array(n_str.length);
    for(var i=0; i<n_str.length ; i++){
        arr[i]=n_str.charAt(i);
    }
    if(arr[0]=='0'){
        for(var i=0; i<arr.length; i++){
            if(arr[i]=='0'){
                arr[i]='';
            }
            else break;
        }
    }
    for(var i=0;i<arr.length;i++){
        answer+=arr[i];
    }
    return answer;
}